mylfs.sh

```diff
@@ -480,39 +501,41 @@ function build_phase {
     fi
 
     PHASE=$1
+    PHASESECTION=lfs
+    if ! [ -e $2 ]; then PHASESECTION=$2; fi
 
     if [ -n "$STARTPHASE" ]
     then
         if [ $PHASE -lt $STARTPHASE ] || { $FOUNDSTARTPHASE && $ONEOFF; }
         then
-            echo "Skipping phase $PHASE"
+            echo "Skipping phase ${_PHASES[$PHASE]} in $PHASESECTION section"
             return 0
         else
             FOUNDSTARTPHASE=true
         fi
     fi
 
-    if [ $PHASE -ne 1 -a ! -f $LFS/root/.phase$((PHASE-1)) ]
+    if [ $PHASE -ne 1 -a ! -f $LFS/root/.$PHASESECTION/.phase$((PHASE-1)) ]
     then
-        echo "ERROR: phases preceeding phase $PHASE have not been built"
+        echo "ERROR: phases preceeding phase ${_PHASES[$PHASE-1]} have not been built"
         return 1
     fi
+    SSTR="Section $PHASESECTION. Phase ${_PHASES[$PHASE-1]}"
+    HOUT="# "
+    HOUT+=$(repeat_echo ${#SSTR} '#';echo)
+    HOUT+="\n# ${SSTR}\n"
+    HOUT+="# $(repeat_echo ${#SSTR} '~';echo)"
+    echo -e $HOUT
```

Comment:
Also: this is a cosmetic change
This will make automatic top and bottom line for text